libs/core/output.ts

```diff
@@ -19,12 +19,6 @@ enum Output {
     ALL = 0x0f
 }
 
-enum OutputType {
```

Comment:
Please do not include unrelated changes in your prs. In this case, it's possible a user project is referring to this for some reason so removing it would break things; if there's no particular reason to remove it, it's better to just leave it in.

Reply:
Ok i got it back